build.gradle

```diff
@@ -1,6 +1,55 @@
-/*
- * This file was generated by the Gradle 'init' task.
- *
- * This is a general purpose Gradle build.
- * Learn how to create Gradle builds at https://guides.gradle.org/creating-new-gradle-builds
- */
+plugins {
+    id 'java'
+}
+//Dependency Management
+repositories {
+    mavenCentral()
+    //flatDir{
+     //   dirs ("lib")
+    //}
+    maven {
+        url "https://www.hec.usace.army.mil/nexus/repository/maven-public/"
+    }
+}
+dependencies {
+    implementation "org.junit.jupiter:junit-jupiter:5.4.2"
+    implementation 'mil.army.usace.hec:hec-dssvue:6.0-SNAPSHOT'
+    implementation 'mil.army.usace.hec:hec-core:6.0-SNAPSHOT'
+    implementation 'mil.army.usace.hec:hec-lib:6.0-SNAPSHOT'
+    implementation 'mil.army.usace.hec:hec-rma:6.0-SNAPSHOT'
+    implementation 'org.jdom:jdom:1.1.3'
+    implementation 'com.rmanet:rmadev:6.0-SNAPSHOT'
+    implementation 'mil.army.usace.hec:hecnf:6.0-SNAPSHOT'
+    implementation 'mil.army.usace.hec:hecnfPluginAPI:6.0-SNAPSHOT'
+    implementation 'mil.army.usace.hec:rtsPluginAPI:3.2.1.459b'
+    implementation 'mil.army.usace.hec:rts:3.2.1.459b'
+    implementation 'mil.army.usace.hec:hecMaps:6.0-SNAPSHOT'
+    implementation 'mil.army.usace.hec:hec-data:6.0-SNAPSHOT'
+    implementation 'mil.army.usace.hec:javaHeclib:7-HZ-win-x64@zip'
+    implementation 'org.apache.commons:commons-lang3:3.11'
+}
+//Testing
+test{
+    useJUnitPlatform()
+    testLogging{
```

Comment:
we can start adding tests at some point, this would be a good thing to think through - what should we test?